### ISSUE TITLE
Not worked with Twisted

### ISSUE THREAD

**Issue by heggi** [arskom/spyne#600]
Example code: http://spyne.io/#inprot=Soap11&outprot=Soap11&s=rpc&tpt=TwistedWebResource&validator=true

Try request wsdl file: `curl http://localhost:8000?wsdl`

Got: 
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/twisted/protocols/basic.py", line 572, in dataReceived
    why = self.lineReceived(line)
  File "/usr/lib/python3.7/site-packages/twisted/web/http.py", line 2099, in lineReceived
    self.allContentReceived()
  File "/usr/lib/python3.7/site-packages/twisted/web/http.py", line 2190, in allContentReceived
    req.requestReceived(command, path, version)
  File "/usr/lib/python3.7/site-packages/twisted/web/http.py", line 917, in requestReceived
    self.process()
--- <exception caught here> ---
  File "/usr/lib/python3.7/site-packages/twisted/web/server.py", line 194, in process
    resrc = self.site.getResourceFor(self)
  File "/usr/lib/python3.7/site-packages/twisted/web/server.py", line 853, in getResourceFor
    return resource.getChildForRequest(self.resource, request)
  File "/usr/lib/python3.7/site-packages/twisted/web/resource.py", line 98, in getChildForRequest
    resource = resource.getChildWithDefault(pathElement, request)
  File "/usr/lib/python3.7/site-packages/spyne/server/twisted/http.py", line 455, in getChildWithDefault
    return get_twisted_child_with_default(self, path, request)
  File "/usr/lib/python3.7/site-packages/spyne/server/twisted/http.py", line 418, in get_twisted_child_with_default
    request.realprepath = '/' + '/'.join(request.prepath)
builtins.TypeError: sequence item 0: expected str instance, bytes found
```
pip list | grep -E 'spyne|Twisted'
spyne                  2.13.9a0  
Twisted                18.9.0

**Comment by heggi** [comment]
Python 3.7.2
Archlinux

**Comment by plq** [comment]
thanks for the bug report! a7cd235 is supposed to fix this. reopen if you disagree

**Comment by daniel-butler** [comment]
This is fixed for python 3.8.0 too with spyne 2.13.11a0
